pom.xml

```diff
@@ -25,10 +25,11 @@
         <eclipselink.version>2.7.0</eclipselink.version>
         <jackson.version>2.9.2</jackson.version>
         <guice.version>4.1.0</guice.version>
-        <guava.version>23.3-jre</guava.version>
+        <guava.version>21.0</guava.version>
```

Comment:
Can we use latest version here `23.4-jre`, it was `23.3-jre`, not sure why we need `21.0`?

Reply:
Fixed